source/includes/fundamentals/automatic-db-coll-creation.rst

```diff
@@ -0,0 +1,6 @@
+.. tip:: Non-existent Databases and Collections
+
+   The driver automatically creates the necessary database and
```

Comment:
The driver does not do anything special on the first write operation. The server creates the collection/database if it does not exist on the first write operation.

```suggestion
   The server automatically creates the necessary database and
```